modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/cluster/init/ClusterInitOptions.java

```diff
@@ -55,22 +68,64 @@ public class ClusterInitOptions {
     @Option(names = {CLUSTER_NAME_OPTION, CLUSTER_NAME_OPTION_SHORT}, required = true, description = CLUSTER_NAME_OPTION_DESC)
     private String clusterName;
 
-    @Mixin
-    private AuthenticationOptions authenticationOptions;
+    @ArgGroup
+    private ClusterConfigOptions clusterConfigOptions;
 
+    private static class ClusterConfigOptions {
+        @Option(names = {CLUSTER_CONFIG_OPTION, CLUSTER_CONFIG_OPTION_SHORT}, description = CLUSTER_CONFIG_OPTION_DESC)
+        private String config;
+
+        @Option(names = {CLUSTER_CONFIG_FILE_OPTION, CLUSTER_CONFIG_FILE_OPTION_SHORT}, description = CLUSTER_CONFIG_FILE_OPTION_DESC)
+        private File file;
+    }
+
+    /**
+     * Consistent IDs of the nodes that will host the Meta Storage.
+     *
+     * @return Meta storage node ids.
+     */
     public List<String> metaStorageNodes() {
         return metaStorageNodes;
     }
 
+    /**
+     * Consistent IDs of the nodes that will host the Cluster Management Group; if empty,
+     * {@code metaStorageNodeIds} will be used to host the CMG as well.
+     *
+     * @return Cluster management node ids.
+     */
     public List<String> cmgNodes() {
         return cmgNodes;
     }
 
+    /**
+     * Human-readable name of the cluster.
+     *
+     * @return Cluster name.
+     */
     public String clusterName() {
         return clusterName;
     }
 
-    public AuthenticationOptions authenticationOptions() {
-        return authenticationOptions;
+    /**
+     * Cluster configuration.
+     *
+     * @return Cluster configuration.
+     */
+    @Nullable
+    public String clusterConfiguration() {
+        if (clusterConfigOptions == null) {
+            return null;
+        } else if (clusterConfigOptions.config != null) {
+            return clusterConfigOptions.config;
+        } else if (clusterConfigOptions.file != null) {
+            try (Stream<String> lines = Files.lines(clusterConfigOptions.file.toPath())) {
```

Comment:
Can't we store `Path` in cluster config options? Why is there a File?
Just a question, you don't have to fix it